main.go

```diff
@@ -154,7 +134,7 @@ func wrapMain() error {
 		}
 	}()
 
-	runner := NewEnviron(filepath.Join(workspace.Path, vargs.Dir), os.Environ(),
+	runner := NewEnviron(filepath.Join(workspace, vargs.Dir), os.Environ(),
```

Comment:
the only property of `plugin.Workspace{}` that was being used was `Path`, so `workspace` is just becoming a string to fit the new built-in drone variables.